lib/streamlit/elements/multiselect.py

```diff
@@ -254,6 +244,46 @@ def _multiselect(
 
         opt = ensure_indexable(options)
 
+        @overload
```

Comment:
This addition seems like the same as line 49. Seems like we can just remove this?